src/libs/ReportPrimaryActionUtils.ts

```diff
@@ -19,21 +28,28 @@ import {
     isSettled,
 } from './ReportUtils';
 import {getSession} from './SessionUtils';
-import {allHavePendingRTERViolation, isDuplicate, isOnHold as isOnHoldTransactionUtils, shouldShowBrokenConnectionViolationForMultipleTransactions} from './TransactionUtils';
-
-function isSubmitAction(report: Report, policy: Policy) {
+import {
+    allHavePendingRTERViolation,
+    hasPendingRTERViolation as hasPendingRTERViolationTransactionUtils,
+    isDuplicate,
+    isOnHold as isOnHoldTransactionUtils,
+    shouldShowBrokenConnectionViolationForMultipleTransactions,
+    shouldShowBrokenConnectionViolation as shouldShowBrokenConnectionViolationTransactionUtils,
+} from './TransactionUtils';
+
+function isSubmitAction(report: Report, policy?: Policy) {
     const isExpenseReport = isExpenseReportUtils(report);
     const isReportSubmitter = isCurrentUserSubmitter(report.reportID);
     const isOpenReport = isOpenReportUtils(report);
     const isManualSubmitEnabled = getCorrectedAutoReportingFrequency(policy) === CONST.POLICY.AUTO_REPORTING_FREQUENCIES.MANUAL;
 
-    return isExpenseReport && isReportSubmitter && isOpenReport && isManualSubmitEnabled;
+    return isExpenseReport && isReportSubmitter && isOpenReport && isManualSubmitEnabled && report.total !== 0;
```

Comment:
I think we should check if the report has expenses, since it's technically possible to submit a report with total = 0 in OldDot

![Screenshot 2025-04-01 at 9 39 45 AM](https://github.com/user-attachments/assets/47458e29-d9e5-487c-a15f-d3012e85f457)

Reply:
Totally agree with that.

Reply:
Thanks for explanation! I've updated accordingly